modules/mobile/spec/request/discovery_request_spec.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require_relative '../support/helpers/spec_helper'
+require_relative '../support/helpers/sis_session_helper'
 require_relative '../support/matchers/json_schema_matcher'
 
-RSpec.describe 'discovery', type: :request do
+RSpec.describe 'discovery', skip_json_api_validation: true, type: :request do
```

Comment:
If we're skipping the validation, does that mean it's currently failing? I assume we can't fix most of them without breaking the schema. So what's the plan for these?

Reply:
yes, all the skipped ones are failing the check. One of the tickets I linked in the summary of the PR is for evaluating these. Many of them probably can be fixed without breaking FE but it'll require a lot of coordination with FE.